README.md

```diff
@@ -1,2 +1,64 @@
-# sdk-js
-Evefan sdk to manage events and query information
+# Evefan SDK for JavaScript
+
+Evefan SDK to manage events and query information
+
+## Installation
+
+Install the SDK using npm:
+
+```
+npm install @evefan/sdk-js
+```
+
+## API Reference
+
+### `EvefanSDK(writeKey, workerUrl)`
+
+Creates a new instance of the Evefan SDK.
+
+- `writeKey`: Your Evefan write key
+- `workerUrl`: URL of the Evefan worker
+
+### `evefan.query(sqlQuery)`
+
+Executes a SQL query against your Evefan data.
+
+- `sqlQuery`: SQL query string
+
+Returns a Promise that resolves with the query results.
+
+## Code Snippet for Node.js
+
+Here's a standalone code snippet you can use as a starting point:
+
+```
+const EvefanSDK = require("@evefan/sdk-js").default;
```

Comment:
I think you can just do import EvefanSDK from '@evefan/sdk-js'. Assuming the file is .cjs and not .js. Lets change it in the docs

Reply:
the generated file is js and had some issues on the import without the default

Reply:
that's because you're using it from a js file that uses require and not import. That's ok to change for the example dw :)

Reply:
Import solved in the last commit